<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpellerDialog</class>
 <widget class="QDialog" name="SpellerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Check Spelling</string>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Unknown word:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditOriginal">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="pushButtonReplace">
     <property name="text">
      <string>Replace</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Replace with:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditNew"/>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="pushButtonIgnore">
     <property name="text">
      <string>Ignore</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>371</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="pushButtonAlwaysIgnore">
     <property name="text">
      <string>Always ignore</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Suggested words :</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" rowspan="2" colspan="2">
    <widget class="QListWidget" name="listWidget">
     <property name="modelColumn">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="3" column="3" rowspan="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>87</width>
       <height>91</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>110</width>
       <height>61</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="pushButtonIgnoreList">
     <property name="text">
      <string>Show Ignore List \/</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="1" colspan="3">
    <widget class="QFrame" name="ignoreListFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="_2">
      <item row="0" column="0">
       <widget class="QListView" name="ignoreListView"/>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="lineEditIgnoredWord"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButtonAddRemove">
        <property name="text">
         <string>Add/Remove</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="labelMessage">
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushButtonReplace</tabstop>
  <tabstop>pushButtonIgnore</tabstop>
  <tabstop>pushButtonAlwaysIgnore</tabstop>
  <tabstop>lineEditOriginal</tabstop>
  <tabstop>lineEditNew</tabstop>
  <tabstop>listWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>SpellerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
